server/src/instant/reactive/session.clj

```diff
@@ -199,7 +201,7 @@
         _ (reset! debug-info {:processed-tx-id processed-tx-id
                               :instaql-queries (map :instaql-query/query stale-queries)})
         recompute-results (->> stale-queries
-                               (ua/vfuture-pmap (partial recompute-instaql-query! opts)))
+                               (ua/pmap-all (partial recompute-instaql-query! opts)))
```

Comment:
Can you put this behind a `instant.flags` in case we run into unexpected issue in production?